RetailCoder.VBE/Navigation/Folders/FolderHelper.cs

```diff
@@ -23,7 +23,7 @@ public CodeExplorerCustomFolderViewModel GetFolderTree(Declaration declaration =
         {
             var delimiter = GetDelimiter();
 
-            var root = new CodeExplorerCustomFolderViewModel(null, string.Empty, string.Empty);
+            var root = new CodeExplorerCustomFolderViewModel(null, string.Empty, string.Empty, _state.ProjectsProvider);
 
             var items = declaration == null
                 ? _state.AllUserDeclarations.ToList()
```

Comment:
Would it be better to get the `Finder` here?

Reply:
I had to pass it in here because the view model newed up in the `Add` method needs it to deal with document modules.

Reply:
No, I meant I think we should eventually deprecate `RubberduckParserState.AllUserDeclarations`, no?

Reply:
Ah, I thought the comment had something to do with the diff.

Yes, that should probably use the finder, like so many other places.